src/gui/MainWindow.cpp

```diff
@@ -597,6 +597,41 @@ void MainWindow::prepareUi() {
     });
     verifyValidateMenu->addAction(verifyValidateViewportAct);
 
+    QAction* verificationValidationNewTest = new QAction(tr("Create new test"), this);
+    verificationValidationNewTest->setIcon(QPixmap::fromImage(coloredIcon(":/icons/verifyValidateIcon.png", "$Color-MenuIconVerifyValidate")));
```

Comment:
maybe new icon for each (create new test, remove test, etc..)

Reply:
No time